quickshop-bukkit/src/main/java/com/ghostchu/quickshop/shop/SimplePriceLimiter.java

```diff
@@ -168,14 +169,36 @@ public PriceLimiterCheckResult check(@NotNull CommandSender sender, @NotNull Ite
                 return new SimplePriceLimiterCheckResult(PriceLimiterStatus.NOT_A_WHOLE_NUMBER, undefinedMin, undefinedMax);
             }
         }
+
+        double minPrice = 0;
+        double maxPrice = 0;
+        boolean hasMaxPrice = false;
+        List<ItemStack> flattenedItems = ItemContainerUtil.flattenContents(itemStack, true, false);
```

Comment:
This could be deemed final for readability purposes.

Reply:
I had assumed this wasn't in the projects style given how other stack variables are formatted in the project. I'll make these changes.

Reply:
> I had assumed this wasn't in the projects style given how other stack variables are formatted in the project. I'll make these changes.

The style guidelines will be transitioning to a more rigid style once I have some time since I took over the project. Thank you for the changes.